ChessBoardApp/ChessBoard/ChessBoardDialog.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.Extensions.Logging;
+
+namespace ChessBoardApp.ChessBoard
+{
+
+    public class ChessBoardDialog
+    {
+
+        public static string EnterRectangleWidth()
+        {
+            Console.Write("Please enter rectangle width: ");
+            return Console.ReadLine();
+        }
+
+        public static string EnterRectangleLength()
+        {
+            Console.Write("Please enter rectangle length: ");
+            return Console.ReadLine();
+        }
+
+        public static void ChessBoardMessage()
```

Comment:
Does not seem to be used, let's clean this up if that's the case

Reply:
Should I leave one general message in the Dialog, for both Rectangle and Square chessboards? Because I had one for Rectangle and one for 4x4 5x5 etc. ChessBoards (so this one, that's not used for the OneLoop, TwoLoop methods)